resources/options_lookup.tsv

```diff
@@ -412,65 +412,65 @@ Insulation Slab	"4ft R5 Under, Horizontal"	ResStockArguments	slab_perimeter_insu
 Insulation Slab	"2ft R5 Perimeter, Vertical"	ResStockArguments	slab_perimeter_insulation_r=5	slab_perimeter_depth=2	slab_under_insulation_r=0	slab_under_width=0	slab_thickness=auto	slab_carpet_fraction=auto	slab_carpet_r=auto																			
 Insulation Slab	"2ft R10 Perimeter, Vertical"	ResStockArguments	slab_perimeter_insulation_r=10	slab_perimeter_depth=2	slab_under_insulation_r=0	slab_under_width=0	slab_thickness=auto	slab_carpet_fraction=auto	slab_carpet_r=auto																			
 Insulation Slab	"R10 Whole Slab, Horizontal"	ResStockArguments	slab_perimeter_insulation_r=0	slab_perimeter_depth=0	slab_under_insulation_r=10	slab_under_width=999	slab_thickness=auto	slab_carpet_fraction=auto	slab_carpet_r=auto																			
-Insulation Floor	None	ResStockArguments	floor_over_foundation_assembly_r=3.1	floor_over_garage_assembly_r=3.1																								
-Insulation Floor	Uninsulated	ResStockArguments	floor_over_foundation_assembly_r=3.1	floor_over_garage_assembly_r=3.1																								
-Insulation Floor	Ceiling R-13	ResStockArguments	floor_over_foundation_assembly_r=15.6	floor_over_garage_assembly_r=15.6																								
-Insulation Floor	Ceiling R-19	ResStockArguments	floor_over_foundation_assembly_r=20.4	floor_over_garage_assembly_r=20.4																								
-Insulation Floor	Ceiling R-30	ResStockArguments	floor_over_foundation_assembly_r=28.1	floor_over_garage_assembly_r=28.1																								
-Insulation Floor	Ceiling R-38	ResStockArguments	floor_over_foundation_assembly_r=32.9	floor_over_garage_assembly_r=32.9																								
+Insulation Floor	None	ResStockArguments	floor_over_foundation_assembly_r=5.3	floor_over_garage_assembly_r=5.3																								

```

Comment:
@aspeake I realize you didn't start this approach, but I would feel more comfortable if we used zeroes for assembly R-values for None options to ensure these values are not accidentally used. This comment applies not just to floors but other surface types with None options. If these assembly R-values _are_ accidentally used for modeling, an assembly R-value of zero will trigger an error and we can then fix the underlying issue. @joseph-robertson, is there any reason we aren't doing this?

Reply:
I can't think of why they'd need to be set to "Uninsulated" values instead of zero.

There would be a "None" option change for:
* `Insulation Ceiling`
* `Insulation Floor`
* others?

@shorowit @aspeake

Reply:
Those are the only two that I see.

Reply:
👍 I'll update